lib/Controller/RoomController.php

```diff
@@ -741,35 +742,63 @@ public function getParticipants(string $token): DataResponse {
 	 *
 	 * @param string $token
 	 * @param string $newParticipant
+	 * @param string $source
 	 * @return DataResponse
 	 */
-	public function addParticipantToRoom(string $token, string $newParticipant): DataResponse {
+	public function addParticipantToRoom(string $token, string $newParticipant, string $source = 'users'): DataResponse {
```

Comment:
We could also now merge the "Invite email to room" in here.
Additionally to `users`, `groups` it would then allow source `emails`.

I guess it makes sense, and also the emails endpoint we have now was never part of a release. Or did you already implement this too heavily @mario @Ivansss

Reply:
Instead of:
```js
$.post(
	OC.linkToOCS('apps/spreed/api/v1/room', 2) + token + '/participants/guests',
	{
		newParticipant: email
	}
)
```
you do:
```js
$.post(
	OC.linkToOCS('apps/spreed/api/v1/room', 2) + token + '/participants',
	{
		newParticipant: email,
		source: 'emails'
	}
)
```


lets do this.

Reply:
Added as a new commit, if anyone of you does not agree, we can simply drop it from this PR.